src/main/java/com/exadel/discount/controller/CityController.java

```diff
@@ -0,0 +1,52 @@
+package com.exadel.discount.controller;
+
+import com.exadel.discount.dto.CityDTO;
+import com.exadel.discount.dto.validation.Create;
+import com.exadel.discount.service.CityService;
+import io.swagger.annotations.ApiOperation;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.constraints.NotNull;
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/countries")
+@RequiredArgsConstructor
+public class CityController {
+    private final CityService cityService;
+
+    @GetMapping("/cities")
```

Comment:
Do you really need this endpoint? For what?

Reply:
> 
> 
> All andpoints from CityController should be moved to the CountryController with path similar to next: `/countries/{countryId}/cities` because city is dependent on country in any case

So, do I need to delete CityController class and move all endpoint to CountryController?

Reply:
Done

Reply:
Deleted